File: Data Structures & Algorithms/remove-element/submission-0.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        tmp=[]
        for i in range(len(nums)):
            if nums[i]==val:
                continue
            tmp.append(nums[i])
        for i in range(len(tmp)):
            nums[i]=tmp[i]
        return len(tmp)

            
         

File: Data Structures & Algorithms/maximum-product-difference-between-two-pairs/submission-0.java
class Solution {
    public int maxProductDifference(int[] nums) {
     ArrayList<Integer> list=new ArrayList<>();
     for(int x:nums)
     {
        list.add(x);
     }
     Integer max=Collections.max(list);
     Integer min=Collections.min(list);
     list.remove(max);
     list.remove(min);
     Integer max2=Collections.max(list);
     Integer min2=Collections.min(list);
     return (max*max2)-(min*min2);
    }
}